lib/streamlit/runtime/metrics_util.py

```diff
@@ -226,38 +275,47 @@ def wrap(*args, **kwargs):
             and len(ctx.tracked_commands)
             < _MAX_TRACKED_COMMANDS  # Prevent too much memory usage
         )
-
-        # Deactivate tracking to prevent calls inside already tracked commands
-        if ctx:
-            ctx.command_tracking_deactivated = True
-
-        exec_start = timer()
-        result = callable(*args, **kwargs)
-
-        # Activate tracking again
-        if ctx:
-            ctx.command_tracking_deactivated = False
-
-        if not tracking_activated:
-            return result
-
+        command_telemetry: Union[Command, None] = None
```

Comment:
Changed it 👍 

There is actually some discussion that using `Union[X, None]` instead of `Optional[X]` is slightly preferred.. but this isn't the strongest argument 😃:
https://twitter.com/tiangolo/status/1531583406651363328